<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapLocation</class>
 <widget class="QMainWindow" name="MapLocation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(114, 132, 135);
	border-width: 2px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(224, 255, 251);
	font: 700 9pt &quot;Segoe UI&quot;;


}</string>
    </property>
    <item>
     <property name="text">
      <string>Cevizlibağ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Davutpaşa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yıldız</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{

	border: 2px;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 2px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(255, 255, 255);
	font: 700 9pt &quot;Segoe UI&quot;;


}</string>
    </property>
    <property name="plainText">
     <string>                               Lokasyonlar  </string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_Cev">
     <widget class="QTextEdit" name="textEdit_Cev">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(114, 132, 135);
	border-width: 2px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(255, 255, 255);
	font: 700 9pt &quot;Segoe UI&quot;;


}</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400;&quot;&gt;Doluluk oranı&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSlotCev_B">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>110</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slot B</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblSlotCev_C">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slot C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblSlotCev_A">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slot A</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarCev">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="value">
       <number>25</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cev_SlotC">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 1px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(178, 213, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
      </property>
      <property name="text">
       <string>Seç</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cev_SlotB">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 1px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(178, 213, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
      </property>
      <property name="text">
       <string>Seç</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cev_SlotA">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 1px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(178, 213, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
      </property>
      <property name="text">
       <string>Seç</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_Yildiz">
     <widget class="QLabel" name="lblSlotYil_C">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slot C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblSlotYil_B">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>110</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slot B</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarYil">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_Yil">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(114, 132, 135);
	border-width: 2px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(255, 255, 255);
	font: 700 9pt &quot;Segoe UI&quot;;


}</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400;&quot;&gt;Doluluk oranı&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSlotYil_A">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slot A</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Yil_SlotA">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 1px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(178, 213, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
      </property>
      <property name="text">
       <string>Seç</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Yil_SlotB">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 1px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(178, 213, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
      </property>
      <property name="text">
       <string>Seç</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Yil_SlotC">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 1px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(178, 213, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
      </property>
      <property name="text">
       <string>Seç</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_Dav">
     <widget class="QLabel" name="lblSlotDav_A">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slot A</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_Dav">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(114, 132, 135);
	border-width: 2px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(255, 255, 255);
	font: 700 9pt &quot;Segoe UI&quot;;


}</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400;&quot;&gt;Doluluk oranı&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSlotDav_C">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slot C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblSlotDav_B">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>110</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slot B</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarDav">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dav_SlotA">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 1px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(178, 213, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
      </property>
      <property name="text">
       <string>Seç</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dav_SlotB">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 1px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(178, 213, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
      </property>
      <property name="text">
       <string>Seç</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dav_SlotC">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
	border-width: 1px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(178, 213, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
      </property>
      <property name="text">
       <string>Seç</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_ShowInMap">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(114, 132, 135);
	border-width: 2px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(224, 255, 251);
	font: 700 9pt &quot;Segoe UI&quot;;


}</string>
    </property>
    <property name="text">
     <string>Haritada Göster</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>0</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ayarlar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kullanıcıyı Sil</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Çıkış</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>791</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_pic</zorder>
   <zorder>comboBox</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>pushButton_ShowInMap</zorder>
   <zorder>comboBox_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
